### PR TITLE
FIX: Make compatible with latest breaking changes

### PR DESCRIPTION
**Pull request by Firepup6500**
This plugin currently breaks user profile pages due to using a depreciated args. This PR makes it target the correct thing according to the depreciation warning.

### CODE REVIEW

**Comment by Firepup6500** [comment]
The warning:
```
Deprecation notice: PluginOutlet arguments should now be passed using `@outletArgs=` instead of `@args=` (outlet: user-profile-avatar-flair) [deprecation id: discourse.plugin-outlet-args]
```

**Comment by Firepup6500** [comment]
There is additionally this depreciation warning that I'm not sure how to fix:
```
Deprecation notice: The `tagName` argument to PluginOutlet is deprecated. If a wrapper element is required, define it manually around the outlet call. Using tagName will prevent wrapper PluginOutlets from functioning correctly. (outlet: user-profile-avatar-flair) [deprecation id: discourse.plugin-outlet-tag-name]
```

**Comment by davidtaylorhq** [comment]
Thanks @Firepup6500. The `tagName` deprecation can be resolved by removing `@tagName=` and wrapping the whole PluginOutlet in a `<div` instead. 

Basically we need to copy how core does it here: https://github.com/discourse/discourse/blob/d5d0bab19d5b807e3068f97a5fc085fb4aa847ff/app/assets/javascripts/discourse/app/components/user-profile-avatar.hbs#L1-L11

**Comment by Firepup6500** [comment]
How's that @davidtaylorhq?

File: assets/javascripts/discourse/templates/components/user-profile-avatar.hbs [hunk 1]
```diff
@@ -2,10 +2,11 @@
...
-  />
-</div>
\ No newline at end of file
+  <div>
+    <PluginOutlet
+      @name="user-profile-avatar-flair"
+      @connectorTagName="div"
+      @outletArgs={{hash model=@user}}
+    />
+    </div>
```
[the remark this answers is not in the document]
Comment (Firepup6500) [reply assets/javascripts/discourse/templates/components/user-profile-avatar.hbs:11]: Oh oops, let me fix that 😅

[the remark this answers is not in the document]
Comment (Firepup6500) [reply assets/javascripts/discourse/templates/components/user-profile-avatar.hbs]: How's that now?

[the remark this answers is not in the document]
Comment (Arkshine) [reply assets/javascripts/discourse/templates/components/user-profile-avatar.hbs]: It should be okay now, thanks. 😄

File: assets/javascripts/discourse/templates/components/user-profile-avatar.hbs [hunk 2]
```diff
@@ -2,10 +2,11 @@
   {{animated-bound-avatar @user "huge"}}
   <UserAvatarFlair @user={{@user}} />
   {{! template-lint-disable no-capital-arguments }}
```
Comment (davidtaylorhq) [inline assets/javascripts/discourse/templates/components/user-profile-avatar.hbs:4]:
```suggestion
```
(`@args` has now been replaced with `@outletArgs`, so this linting error is resolved 🎉)

**Review by davidtaylorhq — APPROVED** [review]
Thanks @Firepup6500 🙌

**Pull request merged**